packages/styles/components/SplitButton.module.scss

```diff
@@ -4,6 +4,10 @@
   display: inline-flex;
   outline: none;
   position: relative;
+
+  button[role="button"] {
```

Comment:
Why we are using `[role="button"]` to get it?

Reply:
just to be more specific hahhaha I don't know if another one will emerge 🤷🏾‍♂️

Reply:
I've change to class here 43e5dfc